parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -208,7 +209,7 @@ ensureRunc() {
 
     TARGET_VERSION=$1
     if [[ -z ${TARGET_VERSION} ]]; then
-        TARGET_VERSION="1.0.0-rc95"
+        TARGET_VERSION="1.0.2"
```

Comment:
1.0.3? 😃

Reply:
Chose runc 1.0.2 based on Brian's email (with the compatibility). Is it ok to change it to 1.0.3?

Reply:
I think `1.0.3` should also be ok as it's the latest version and it also seems to be used by default in the node (and I didn't see errors for runc when checking).

Reply:
yeah I think it's fine. I do think we need to bump the 1.4.xx containerd branch to 1.14.12 as well, I believe upgrading runc without that broke moby install because there's now a conflict with previous version of runc and moby-containerd 1.14.12: https://github.com/Azure/AgentBaker/commit/cca3a2cfe8655c0c1753e294173cd9b38bdc9bc4

applies to both docker and containerd I think: https://github.com/Azure/AgentBaker/blob/844736ca0cf105beb6238e2441c0bd8ebd7ecc59/parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh#L189
https://github.com/Azure/AgentBaker/blob/844736ca0cf105beb6238e2441c0bd8ebd7ecc59/parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh#L144-L145


1.14.12-2 I believe

```
azureuser@ace-vm:~/go/src/goms.io/aks/rp$ apt-cache madison moby-containerd
moby-containerd | 1.4.12+azure-2 | https://packages.microsoft.com/ubuntu/18.04/multiarch/prod bionic/main amd64 Packages
moby-containerd | 1.4.12+azure-1 | https://packages.microsoft.com/ubuntu/18.04/multiarch/prod bionic/main amd64 Packages
moby-containerd | 1.4.11+azure-1 | https://packages.microsoft.com/ubuntu/18.04/multiarch/prod bionic/main amd64 Packages
```

Reply:
Thanks for the pretty detailed and helpful comment. I've updated the PR and just created a standalone environment build to test it.